specification/apimanagement/resource-manager/readme.md

```diff
@@ -488,6 +488,7 @@ swagger-to-sdk:
     after_scripts:
       - bundle install && rake arm:regen_all_profiles['azure_mgmt_api_management']
   - repo: azure-resource-manager-schemas
+  - repo: azure-cli
```

Comment:
Hi @DreamlessA , would you please don't add this "- repo: azure-cli"? Currently it will take no action